Sources/Host/Linux/ARM64/PTraceARM64.cpp

```diff
@@ -60,20 +60,11 @@ int PTrace::getMaxHardwareWatchpoints(ProcessThreadId const &ptid) {
 ErrorCode PTrace::readCPUState(ProcessThreadId const &ptid,
                                ProcessInfo const &pinfo,
                                Architecture::CPUState &state) {
-  pid_t pid;
-
-  ErrorCode error = ptidToPid(ptid, pid);
```

Comment:
Yeah I think that's when I was doing the initial support for aarch64, I used the ARM version as a template and never realized that the pid wasn't used in the ARM64 version.